.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "public/themes"]
 	path = public/themes
 	url = git://github.com/openSUSE/opensuse-themes.git
+
+[submodule "public/chameleon"]
+	path = public/chameleon
+	url = git@github.com:openSUSE-zh/chameleon.git
```

Comment:
There's already a opensuse-themes project. Would it make sense to merge it there?